CHANGELOG.md

```diff
@@ -105,3 +108,4 @@ See related PR: [moby/moby#35812](https://github.com/moby/moby/pull/35812)
 + Add Packaging for Fedora 27 [docker/docker-ce-packaging#59](https://github.com/docker/docker-ce-packaging/pull/59)
 * Change default versioning scheme to 0.0.0-dev unless specified for packaging [docker/docker-ce-packaging#67](https://github.com/docker/docker-ce-packaging/pull/67)
 * Pass Version to engine static builds [docker/docker-ce-packaging#70](https://github.com/docker/docker-ce-packaging/pull/70)
++ Added support for aarch64 on Debian (stretch/jessie) and Ubuntu Xenial or newer [docker/docker-ce-packaging#35](https://github.com/docker/docker-ce-packaging/pull/35)
```

Comment:
There was already `aarch64` support for Ubuntu Xenial, this should be for Ubuntu **Zesty** instead.